kolibri/utils/cli.py

```diff
@@ -14,9 +14,9 @@
 
 import kolibri  # noqa
 from kolibri import dist as kolibri_dist  # noqa
-sys.path = [
+sys.path = sys.path + [
     os.path.realpath(os.path.dirname(kolibri_dist.__file__))
-] + sys.path
+]
```

Comment:
Yup, I originally used `sys.append` but didn't think further about it when it was changed.

What's your take on it? I don't like messing with the sys.path at all.. at least only in an non-intrusive way where we append our path.